playerbot/strategy/values/PossibleTargetsValue.h

```diff
@@ -11,16 +11,19 @@ namespace ai
         PossibleTargetsValue(PlayerbotAI* ai, string name = "possible targets", float range = sPlayerbotAIConfig.sightDistance, bool ignoreLos = false) :
           NearestUnitsValue(ai, name, range, ignoreLos) {}
 
+    public:
+        static void FindPossibleTargets(Player* player, list<Unit*>& targets, float range);
+        static bool IsValid(Player* player, Unit* target, float range);

```

Comment:
Make this static so we can call it from AttackersValue (we need to check the possible targets from the given player)